pages/help/glossary.mdx

```diff
@@ -0,0 +1,91 @@
+## General 
+
+| Acronym/Abbreviation         | Definition                                                                               |
+| ---------------------------- | ---------------------------------------------------------------------------------------- |
+| ACL                          | Access Control List                                                                      |
+| API                          | Application Programming Interface                                                        |
+| ARPA                         |  Accountable, Responsible, Participant, Advisor                                          |
+| BAM                          | Bidirectional Associative Memory                                                         |
+| BCR                          | Biospecimen Core Resource                                                                |
+| CI/CD                        |                                                                                          |
+| CNV                          |  Copy Number Variation                                                                   |
+| DCA                          | Data Curator App                                                                         |
+| DCC                          | Data Coordinating Center                                                                 |
+| eRA                          | Electronic Research Administration                                                       |
+| FISH                         |                                                                                          |
```

Comment:
Should be moved to Assays. Would be surprised that there are any who don't know what FISH is, though.